lab-anthony/app/html/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html ng-app="monsterMash">
+  <head>
+    <meta charset="utf-8">
+    <title>MONSTER MASH</title>
+    <link rel="stylesheet" type="text/css" href="bundle.css">
+    <script src="bundle.js"></script>
+  </head>
+  <body ng-controller="MonsterController as monCtr">
+    <main ng-show="!monCtr.gameOver && monCtr.gameStart">
+      <form nonvalidate name="gameForm">
+        <select ng-model="monCtr.gameForm.direction">
+            <option value="east">east</option>
+            <option value="west">west</option>
+          </select>
+          <input type="button" ng-click="monCtr.moveDirection(monCtr.gameForm.direction)" value="Move">
+          <input type="button" ng-disabled="!monCtr.room.monster" ng-click="monCtr.hurtMonster()" value="Hurt Monster">
+      </form>
+      <section>
+        <h2>Stats: </h2>
+        <pre>
+          {{monCtr.player.hp}}
```

Comment:
A number shows up here, but there isn't any label that tells the user what that number means.